docs/reference/cli/rack.md

```diff
@@ -53,6 +51,58 @@ Get logs for the rack
     2020-02-10T13:38:04Z service/web/a55eb25e-90f5-4301-99fd-e35c91128592 id=f492a0dce931 ns=api at=SystemGet method="GET" path="/system" response=200 elapsed=332.219
     ...
 
+## rack mv
+
+Move locally installed Racks to be managed by a organization within a Console installation, or move an organization's Rack back down to be managed locally.
```

Comment:
Is there a way to say this more simply that still captures the subtlety?

"Locally installed" sounds like the Rack is running on my machine

"Console installation" is a bit overloaded

Is there a single short sentence that describes the effect of the change?